domain/src/test/java/org/oppia/android/domain/classify/rules/numericinput/NumericInputEqualsRuleClassifierProviderTest.kt

```diff
@@ -67,13 +77,13 @@ class NumericInputEqualsRuleClassifierProviderTest {
   @Test
   fun testPositiveRealAnswer_positiveRealInput_valuesInRange_bothValuesMatch() {
     val inputs = mapOf(
-      "x" to createReal(
+      "x" to InteractionObjectTestBuilder.createReal(
         value = 5 * FLOAT_EQUALITY_INTERVAL
       )
     )
 
     val matches = inputEqualsRuleClassifier.matches(
-      answer = createReal(value = 5 * FLOAT_EQUALITY_INTERVAL + FLOAT_EQUALITY_INTERVAL / 10),
+      answer = InteractionObjectTestBuilder.createReal(value = 5 * FLOAT_EQUALITY_INTERVAL + FLOAT_EQUALITY_INTERVAL / 10),
```

Comment:
This line has exceeded max line length (100) . For avoiding this kind of issue, try to keep your code behind the grey line that is shown in Android Studio. Refer to KTlint Wiki for more details